app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -246,11 +252,16 @@ export default {
       }
     },
 
-    async updateIngredient(res) {
-      const ingredientEdited = { id: this.ingredientToEdit.id, ...JSON.parse(res.config.data).ingredient };
+    async updateIngredient(ingredientEdited) {
+      ingredientEdited.quantity = ingredientEdited.ingredient_measures_attributes[0].quantity;
+      ingredientEdited.measure = ingredientEdited.ingredient_measures_attributes[0].name;
+      ingredientEdited.id = this.ingredientToEdit.id;
+      ingredientEdited.otherMeasures = { data: [] };
+      ingredientEdited.ingredient_measures_attributes.forEach(element => {
+        ingredientEdited.otherMeasures.data.push({ id: element.id, attributes: { name: element.name, quantity: element.quantity } })
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 132. Maximum allowed is 120             vue/max-len